api/controlplane/v1beta1/k0s_types.go

```diff
@@ -45,7 +45,8 @@ type K0sControlPlaneSpec struct {
 	MachineTemplate *K0sControlPlaneMachineTemplate `json:"machineTemplate"`
 	//+kubebuilder:validation:Optional
 	//+kubebuilder:default=1
-	Replicas int32 `json:"replicas,omitempty"`
+	Replicas int32  `json:"replicas,omitempty"`
+	Version  string `json:"version,omitempty"`
```

Comment:
That feels weird to have both `version` and `k0sVersion` on the same object. Can we just remove `k0sVersion`?
Basically, we can support the k0s and k8s versions here, adding `+k0s.0` if it's missing. @jnummelin wdyt?

Reply:
I removed `K0sVersion` field